espnet2/gan_tts/hifigan/hifigan.py

```diff
@@ -566,7 +566,7 @@ def apply_spectral_norm(self):
         """Apply spectral normalization module from all of the layers."""
 
         def _apply_spectral_norm(m: torch.nn.Module):
-            if isinstance(m, torch.nn.Conv2d):
+            if isinstance(m, torch.nn.Conv1d):
```

Comment:
https://github.com/espnet/espnet/issues/5215 Hi @jerryuhoo , could you double check the issue?